openshift_day2_configurator/configurators/nodes.py

```diff
@@ -0,0 +1,611 @@
+                        "networkd": {},
+                        "passwd": {},
+                        "storage": {
+                            "files": [
+                                {
+                                    "contents": {
+                                        "source": f"data:text/plain;charset=utf-8;base64,{str_b64encode(str_to_encode=chrony_conf)}"
+                                    },
+                                    "mode": 420,
+                                    "overwrite": True,
+                                    "path": "/etc/chrony.conf",
+                                }
+                            ]
+                        },
+                    }
+                },
+            ),
+            logger=logger,
+        )
+    }
+
+    if not (
+        chrony_updated_dict := wait_for_machine_config_pool_to_update(
+            client=client,
+            nodes_type=nodes_type,
+            machine_config_name=machine_config_name,
+            node_operation_message=chrony_desc_message,
+            error_message=f"Failed to Configure chrony NTP on {nodes_type} nodes",
+        )
+    )[chrony_desc_message]["res"]:
+        return chrony_updated_dict
+
+    logger.info(f"Configured chrony NTP on {nodes_type} nodes successfully.")
+
+    return chrony_ntp_task_dict
+
+
+def adding_kernel_arguments_to_nodes(
+    client: DynamicClient,
+    logger: logging.Logger,
+    nodes_type: str,
+) -> Dict[str, Dict[str, Union[str, bool]]]:
+    kernel_argument_desc_message: str = adding_kernel_arguments_to_nodes_message(nodes_type=nodes_type)
+    logger.debug(kernel_argument_desc_message)
+
+    new_kernel_argument = "enforcing=0"
+
+    machine_config_name = f"05-{nodes_type}-kernelarg-selinuxpermissive"
+
+    kernel_argument_task_dict = {
+        kernel_argument_desc_message: create_ocp_resource(
+            ocp_resource=MachineConfig(
+                client=client,
+                name=machine_config_name,
+                label={MACHINE_CONFIGURATION_ROLE_LABEL: nodes_type},
+                config={
+                    "ignition": {
+                        "version": IGNITION_VERSION,
+                    }
+                },
+                kernel_arguments=[new_kernel_argument],
+            ),
+            logger=logger,
+        )
+    }
+
+    if not (
+        kernel_args_updated_dict := wait_for_machine_config_pool_to_update(
+            client=client,
+            nodes_type=nodes_type,
+            machine_config_name=machine_config_name,
+            node_operation_message=kernel_argument_desc_message,
+            error_message=f"Failed to add {new_kernel_argument} kernel argument to {nodes_type} nodes",
+        )
+    )[kernel_argument_desc_message]["res"]:
+        return kernel_args_updated_dict
+
+    logger.info(f"Added {new_kernel_argument} kernel argument to {nodes_type} nodes successfully.")
+
+    return kernel_argument_task_dict
```

Comment:
```suggestion
   kernel_args_updated_dict = wait_for_machine_config_pool_to_update(
            client=client,
            nodes_type=nodes_type,
            machine_config_name=machine_config_name,
            node_operation_message=kernel_argument_desc_message,
            error_message=f"Failed to add {new_kernel_argument} kernel argument to {nodes_type} nodes",
        )
    )
    if ...wait_for_machine_config_pool_to_update:
    

        logger.info(f"Added {new_kernel_argument} kernel argument to {nodes_type} nodes successfully.")

    return kernel_argument_task_dict
```

Reply:
Not sure I understand this comment; @rnetser you meant to check positive condition instead?

Reply:
yes

Reply:
done